react/features/base/conference/actions.js

```diff
@@ -359,6 +360,26 @@ export function _setAudioOnlyVideoMuted(muted: boolean) {
 }
 
 /**
+ * Action to set whether or not the currently displayed large video is in
+ * high-definition. Will no-op if there is no update to the known resolution.
+ *
+ * @param {boolean} isLargeVideoHD - True if the large video is high-definition.
+ * @returns {Function}
+ */
+export function setLargeVideoHDStatus(isLargeVideoHD) {
+    return (dispatch, getState) => {
+        const conference = getState()['features/base/conference'];
+
+        if (conference.isLargeVideoHD !== isLargeVideoHD) {
```

Comment:
I think we don't need this check because the `set` function from base/redux/functions.js will check this anyway. Am I wrong?

Sorry if I'm missing something!